webtool/fetch.py

```
@@ -244,10 +252,10 @@ def check_url(url: str) -> str:
     url = (url or "").strip()
     u = urlparse(url)
     if u.scheme != "https":
-        raise ValueError(f"nur https-URLs werden unterstützt: {url!r}")
+        raise Vorbedingung(f"nur https-URLs werden unterstützt: {url!r}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)